eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -760,7 +760,7 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
 
     case Event(commit: CommitSig, d: DATA_NORMAL) =>
       Commitments.receiveCommit(d.commitments, commit, keyManager) match {
-        case Success((commitments1, revocation)) =>
+        case Success((commitments1: Commitments, revocation)) =>
```

Comment:
@t-bast this is frankly even weirder

Reply:
Yeah I really don't get why it thinks it's not obvious that this is a `Commitments`...

Reply:
Nevermind, this isn't necessary, removed with 8c6302a.